### ISSUE TITLE
Ability to use the lib as a Python module returning a ULID

### ISSUE THREAD

**Issue by davidbgk** [mdomke/python-ulid#2]
Like so:

```shell
$ python -m ulid
01BTGNYV6HRNK8K8VKZASZCFPE
```

I guess implementing a `__main__` which returns `ULID.from_datetime(datetime.now())` by default.

It would ease the generation when calling from a Makefile for instance. My current alternative is the less elegant:

```shell
$ python -c 'from ulid import ULID;print(str(ULID()))'
```

**Comment by mdomke** [comment]
Hi @davidbgk, thanks for the good idea (that you had long, long time ago). It is now implemented in `python-ulid` `2.0.1`

**Comment by davidbgk** [comment]
Oh thanks for the notification! For future reference, it happened in https://github.com/mdomke/python-ulid/commit/ffe282ae029092ab89419010818ffa1c37171592
